src/map/packets.h

```diff
@@ -2942,13 +2942,14 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x08A4,36,clif->pStoragePassword,0);
 	packet(0x0363,8,clif->pDull); // CZ_JOIN_BATTLE_FIELD
 	packet(0x0436,4,clif->pDull); // CZ_GANGSI_RANK
+	packet(0x09ea,11,clif->pRodexReadMail); // CZ_REQ_READ_RODEX
```

Comment:
Was in previous review already
This line in duplicated for line 2952.
This is non shuffle packets and cant be added into shuffle packets.
In line 2952 it added already